gulpfile.js

```diff
@@ -216,6 +217,24 @@ gulp.task(
   })
 );
 
+gulp.task('build-html', () => {
+  const clientIdRequiredMsg =
+    'You must configure an OAuth client ID with the "OAUTH_CLIENT_ID" env var';
+  const apiUrl = process.env.API_URL || 'https://hypothes.is/api/';
+  const config = {
+    apiUrl,
+    authDomain: url.parse(apiUrl).hostname,
+
+    oauthClientId: process.env.OAUTH_CLIENT_ID || clientIdRequiredMsg,
+  };
+  const htmlSafeJsonConfig = JSON.stringify(config).replace(/</g, '\\u003c');
```

Comment:
Even though it is unlikely that the JSON configuration will be unsafe for naive insertion into HTML, I think we should always do the proper safe serialization just in case.